doc/classes/EditorInterface.xml

```diff
@@ -96,6 +96,12 @@
 				Returns the edited (current) scene's root [Node].
 			</description>
 		</method>
+		<method name="get_editor_language" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns the language used for the editor interface.
```

Comment:
I assume this is the language _code_? Or is it actually the _locale_? 😵

Reply:
It's the locale. (Strings like `zh_CN` and `es_AR` might be returned.)

"language" is used since it comes from `interface/editor/editor_language` and language codes are valid locales.